pkg/cmd/server/origin/master_config.go

```diff
@@ -635,7 +635,8 @@ func newAdmissionChain(pluginNames []string, admissionConfigFilename string, plu
 
 		case serviceadmit.RestrictedEndpointsPluginName:
 			// we need to set some customer parameters, so create by hand
-			restrictedNetworks, err := serviceadmit.ParseSimpleCIDRRules([]string{options.NetworkConfig.ClusterNetworkCIDR, options.NetworkConfig.ServiceNetworkCIDR})
+			// PLACEHOLDER HAVE TO FIGURE OUT OUT TO DO
+			restrictedNetworks, err := serviceadmit.ParseSimpleCIDRRules([]string{options.NetworkConfig.ServiceNetworkCIDR})
```

Comment:
You need to pass an array containing all of the clusternetwork cidr values and the servicenetwork cidr value